omnistrate_tests/test_update_memory.py

```
@@ -3,7 +3,7 @@
 from random import randbytes
 from pathlib import Path  # if you haven't already done so
 import threading
-
+from redis.exceptions import (ReadOnlyError)
```

Ruff

`redis.exceptions.ReadOnlyError` imported but unused

Remove unused import: `redis.exceptions.ReadOnlyError`

(F401)